PLIPs/plip-1.md

```diff
@@ -0,0 +1,218 @@
+  - **Networking** - improvements around the communication layer for plasma implementations.
+  - **Interface** - improvements around client [API/RPC] specifications and standards, and also certain language-level standards like method names and contract APIs.
+  - **PRC** - application-level standards and conventions, including contract standards such as token standards (eg. minting logic), name registries, library/package formats, and wallet formats.
+- A **Meta PLIP** describes a process surrounding Plasma or proposes a change to (or an event in) a process. Process PLIPs are like Standards Track PLIPs but apply to areas other than the Plasma protocol itself. They may propose an implementation, but not to Plasma's codebase; they often require community consensus; unlike Informational PLIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Plasma development. Any meta-PLIP is also considered a Process PLIP.
+- An **Informational PLIP** describes an Plasma design issue, or provides general guidelines or information to the Plasma community, but does not propose a new feature. Informational PLIPs do not necessarily represent Plasma community consensus or a recommendation, so users and implementers are free to ignore Informational PLIPs or follow their advice.
+
+It is highly recommended that a single PLIP contain a single key proposal or new idea. The more focused the PLIP, the more successful it tends to be. A change to one client doesn't require a PLIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
+
+A PLIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+
+## PLIP Work Flow
+
+Parties involved in the process are you, the champion or *PLIP author*, the [*PLIP editors*](#plip-editors), and the affected plasma community.
+
+:warning: Before you begin, vet your idea, this will save you time. Ask the Plasma community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Plasma is used. Examples of appropriate public forums to gauge interest around your PLIP include [the plasma.build forum], and [the Issues section of this repository]. In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your PLIP.
```

Comment:
```suggestion
:warning: Before you begin, vet your idea, this will save you time. Ask the Plasma community first if an idea is original to avoid wasting time on something that will be rejected based on prior research (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas where Plasma is used. Examples of appropriate public forums to gauge interest around your PLIP include [the plasma.build forum](https://plasma.build), and [the Issues section of this repository](https://github.com/plasma-group/PLIPs/issues). In particular, [the Issues section of this repository] is an excellent place to discuss your proposal with the community and start creating more formalized language around your PLIP.
```